lasagne/nonlinearities.py

```diff
@@ -1,19 +1,79 @@
 """
-Nonlinearities
+Non-linear activation functions for artificial neurons.
 """
 
+import theano.tensor.nnet
+
+
 # sigmoid
-from theano.tensor.nnet import sigmoid
+def sigmoid(x):
+    """Sigmoid activation function :math:`\\varphi(x) = \\frac{1}{1 + e^{-x}}`
+
+    Parameters
+    ----------
+    x : float32
+        The activation (the summed, weigthed input of a neuron).
+
+    Returns
+    -------
+    float32 in [0, 1]
+        The output of the sigmoid function applied to the activation.
```

Comment:
The range is useful extra information, but [according to the specifications](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id6), that first line should only hold the type (`scalar or tensor`). I would include the range in the description instead.